test/module.test.js

```diff
@@ -42,13 +42,25 @@ describe('basic', () => {
         tagName: 'link',
         rel: 'alternate',
         href: 'nuxt-app.localhost/',
-        hreflang: 'en-US'
+        hreflang: 'en'
       },
       {
         tagName: 'link',
         rel: 'alternate',
         href: 'nuxt-app.localhost/fr',
         hreflang: 'fr-FR'
+      },
+      {
+        tagName: 'link',
+        rel: 'alternate',
+        href: 'nuxt-app.localhost/',
+        hreflang: 'en'
```

Comment:
So now there is a duplicate tag for 'en'? If so, it sounds like a bug.

Reply:
Whoops, should just be a false positive because the two objects are the same. Will update the test and assure it reflects the correct changes

Reply:
Was an FP ☺️